app/views/no_vat_repayments.scala.html

```diff
@@ -16,17 +16,32 @@
 
 @import config.ViewConfig
 @import uk.gov.hmrc.play.views.html._
+@import model.des.BankDetails
 
 @this(
 mainTemplate: views.html.main_template
 )
 
-@()(implicit request: Request[_], messages: Messages, appConfig: ViewConfig)
+@(showBankDetails: Boolean, bankDetails: Option[BankDetails])(implicit request: Request[_], messages: Messages, appConfig: ViewConfig)
```

Comment:
Please make line breaks. The conention to use is:

```
@(
  journeyId: JourneyId,          //some params
  countries: Seq[uk.gov.hmrc.payment.domain.barclays.Country])( // parentheses here
  implicit //implicit divides parameter lists
  request: play.api.mvc.Request[_],  //now implicit parameters
  messages: Messages
)  //closing parenth
```

Reply:
Done